.github/workflows/automated_release.yml

```diff
@@ -15,6 +15,13 @@ jobs:
       contents: write
       packages: write
```

Comment:
I have removed `packages: write` from other repo as it is not required, if this was added for `snok/container-retention-policy@v2` it can be removed.